### DISCUSSION TITLE
Multiple domains

### DISCUSSION THREAD

**Discussion by proudparrot2** [plausible/analytics#3874] [Self-Hosted Support] [answered]
My website is hosted on multiple domains, and how my website functions is users can point their own domains to my server and it will automatically create an SSL certificate for that domain and serve my website.

Are analytics only collected on the domain listed in Plausible (maybe for security reasons or something?), or would it collect on all domains?

**Comment by ruslandoga** [comment, marked as answer]
👋 @proudparrot2 

It would collect on all domains.

**Reply by srubin87** [reply]
I'm not sure this is correct. I have several websites with checkout been made on a different domain. the verification doesn't work when the checkout domain is sending the data

**Reply by ruslandoga** [reply]
👋 @srubin87 

Please share more information about your setup, what "checkout" is, domains involved, browser console logs when sending the data fails, etc.
